primitives/runtime/src/lib.rs

```diff
@@ -902,6 +902,27 @@ impl<R> TransactionOutcome<R> {
 	}
 }
 
+/// Statistics of a block returned by the `dev_getBlockStats` RPC.
+#[derive(Eq, PartialEq, Clone, Copy, Encode, Decode, Debug, TypeInfo)]
+#[cfg_attr(feature = "std", derive(Serialize, Deserialize))]
+#[cfg_attr(feature = "std", serde(rename_all = "camelCase"))]
+pub struct BlockStats {
```

Comment:
And why is that defined in `sp-runtime`? Can that type not just be defined alongside the rpc api declaration?

Reply:
I am totally lost where to define that. If you think it should be beside the RPC I can move it there. I just saw that no type was defined there and figured that this isn't the place.

Reply:
This type doesn't make any sense outside of this rpc. So, we should define it near to the rpc api.

Reply:
Moved it.